src/test/resources/processor_test_files/S4065_ThreadLocalWithInitial/ThreadLocalInitial2.java

```diff
@@ -0,0 +1,14 @@
+
+public class ThreadLocalInitial2 {
```

Comment:
We can rename this also.

```suggestion
public class ThreadLocalInitialWithMultipleStatements {
```